src/index.html

```diff
@@ -11,6 +11,10 @@
       rel="stylesheet"
       href="./styles/main.scss"
```

Comment:
It seems like you're trying to include an SCSS file directly into HTML. Browsers can't interpret SCSS files directly. You should compile the SCSS to CSS and include the compiled CSS file instead.